test/rdoc/test_rdoc_class_module.rb

```diff
@@ -1466,66 +1466,6 @@ def test_update_includes_with_colons
     assert_equal [a, c], @c1.includes
   end
 
-  def test_update_extends
-    a = RDoc::Extend.new 'M1', nil
-    b = RDoc::Extend.new 'M2', nil
-    c = RDoc::Extend.new 'C', nil
-
-    @c1.add_extend a
-    @c1.add_extend b
-    @c1.add_extend c
-    @c1.each_extend do |extend| extend.module end # cache extended modules
-
-    @m1_m2.document_self = nil
-    assert @m1_m2.remove_from_documentation?
```

Comment:
These same test assertions actually appear in other tests like `test_update_includes_with_colons` as well. So we don't need to keep them here either.